app/views/partials/_template_component_question_menu.html.erb

```diff
@@ -0,0 +1,21 @@
+<% @page.components.each do |component| %>
+  <script type="text/html"
+          data-component-template="ComponentQuestionMenu_<%= component.type %>"
+          data-component-uuid="<%= component.uuid %>"
```

Comment:
We probably don't need both of these - if we have the uuid, we don't really need the type.  I think we can probably manage with
```
data-component-template="ComponentQuestionMenu_<%= component.uuid -%>"
```
But I reckon leave it as it is for this PR and I can tweak if needed.